service/dap/config.go

```diff
@@ -27,11 +27,6 @@ func configureSet(sargs *launchAttachArgs, args string) (bool, string, error) {
 		return false, "", fmt.Errorf("%q is not a configuration parameter", cfgname)
 	}
 
-	// If there were no arguments provided, just list the value.
```

Comment:
This is an unfortunate change. Adding -list is much less intuitive and more typing for the user.